### DISCUSSION TITLE
Release Tail length always reverts to Max

### DISCUSSION THREAD

**Discussion by newVideophile** [GrandOrgue/grandorgue#1159] [Help] [answered]
I use the convolution reverb with GrandOrgue v3.7.0-1 on Windows 10. Therefore, in the menu bar, I set the Release Tail Length to the smallest possible value, i.e. 50 ms (I seems that there is no option to completely remove it, which I would prefer). 

However, each time I restart GrandOrgue, this setting is lost and GrandOrgue reverts to "Max", even if I save parameters. 

How to solve this issue ?

**Comment by willeke1234** [comment, marked as answer]
I tried this on macOS, and got stuck at 50 ms. I think it's a bug, it looks like the Release tail length is saved when a sample set is loaded. What happens when you try these steps:

1. Set the Release tail length to 50 ms
2. Load a sample set
3. Restart GrandOrgue

**Comment by newVideophile** [comment]
Yes, it worked, thank you ! 

Still, I agree that it looks like a bug...
